File: examples/bench/string_concat.js
'use strict';
const { performance } = require('perf_hooks');

const N = 20000;
const t0 = performance.now();
let s = '';
for (let i = 0; i < N; i++) {
  s = s + 'x';
}
const t1 = performance.now();
console.log('INNER_MS=' + Math.round(t1 - t0));
console.log('LEN=' + s.length);

File: examples/bench/string_concat.py
import time

N = 20_000
t0 = time.perf_counter()
s = ""
for i in range(N):
    s = s + "x"
t1 = time.perf_counter()
print("INNER_MS=" + str(int(round((t1 - t0) * 1000))))
print("LEN=" + str(len(s)))
